### ISSUE TITLE
Path-mapping issue

### ISSUE THREAD

**Issue by aksyonov** [ReactiveX/rxjs#2988]
**RxJS version:**
5.5.0
Angular-cli: 1.4.9

**Code to reproduce:**
```import {startWith} from 'rxjs/operators/startWith'```

**Expected behavior:**
Should import without errors

**Actual behavior:**
```node_modules/rxjs/_esm5/operators/index.js/startWith.js doesn't exist```
Note `index.js` in path

**Additional information:**
The issue is probably related to `rxjs/_esm5/path-mapping.js` file:
```
...
    "rxjs/operators/groupBy": path.resolve(PATH_REPLACEMENT, "rxjs/_esm5/operators/groupBy.js"),
    "rxjs/operators/ignoreElements": path.resolve(PATH_REPLACEMENT, "rxjs/_esm5/operators/ignoreElements.js"),
    "rxjs/operators/index": path.resolve(PATH_REPLACEMENT, "rxjs/_esm5/operators/index.js"),
    "rxjs/operators": path.resolve(PATH_REPLACEMENT, "rxjs/_esm5/operators/index.js"),
    "rxjs/operators/isEmpty": path.resolve(PATH_REPLACEMENT, "rxjs/_esm5/operators/isEmpty.js"),
    "rxjs/operators/last": path.resolve(PATH_REPLACEMENT, "rxjs/_esm5/operators/last.js"),
    "rxjs/operators/map": path.resolve(PATH_REPLACEMENT, "rxjs/_esm5/operators/map.js"),
    "rxjs/operators/mapTo": path.resolve(PATH_REPLACEMENT, "rxjs/_esm5/operators/mapTo.js"),
    "rxjs/operators/materialize": path.resolve(PATH_REPLACEMENT, "rxjs/_esm5/operators/materialize.js"),
...
```

As you see, `"rxjs/operators"` is between other modules. Importing modules that are before this line works as expected, but all modules after it fail with error. I think need to move line
```
"rxjs/operators": path.resolve(PATH_REPLACEMENT, "rxjs/_esm5/operators/index.js"),
```
after other `"rxjs/operators/..."` modules.

**Comment by DovyByko** [comment]
Same problem. It tries to load operators.js and there isn't a file like that.

**Comment by jasonaden** [comment]
@aksyonov Thanks. I also found this issue. It's due to a problem with how Webpack handles aliasing. Here's the PR that fixes it for rxjs: #2987. I also filed an issue against Webpack as depending on sequence of keys in an object is not good.

**Comment by slawojstanislawski** [comment]
The fix landed in 5.5.2 (https://github.com/ReactiveX/rxjs/commit/24854cf66172f91f55729765c4d64e61607563c8#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR7). Could this be closed then?
